src/backend/distributed/executor/multi_utility.c

```diff
@@ -1082,7 +1084,7 @@ ExecuteCommandOnWorkerShards(Oid relationId, const char *commandString,
 		char *escapedCommandString = quote_literal_cstr(commandString);
 		StringInfo applyCommand = makeStringInfo();
 		appendStringInfo(applyCommand, WORKER_APPLY_SHARD_DDL_COMMAND, shardId,
-						 escapedCommandString);
+						 quote_literal_cstr(schemaName), escapedCommandString);
```

Comment:
Instead of inline quotation, could you this before the for loop?

char *schemaName = get_namespace_name(schemaId);
char *escapedSchemaName = quote_literal_cstr(schemaName);

...